README.md

```diff
@@ -58,36 +63,51 @@ ansible_ssh_pass=vargrant
 ansible_private_key_file=./vm-testing/images/rhel9-vm/.vagrant/machines/trustification/libvirt/private_key
 ```
 
-2. Create  `ansible.cfg` file  in the project with a single VM in the `trustification` group:
+2. Create `ansible.cfg` file in the project with a single VM in the `trustification` group:
+
 ```
 [defaults]
 inventory = ./inventory.ini
 host_key_checking = False
 ```
-3. Add your RH credentials, choose one:
- - as a env variables `SUBSCRIPTION_USERNAME` and `SUBSCRIPTION_PASSWORD` 
- - in the Vagrant file `config.registration.username` and `config.registration.password`
- - during the vagrant up in the registration prompt
 
+3. Add the subscription, registry and certificates information :
+
+- For Red Hat subscription define :
+  export TPA_SINGLE_NODE_REGISTRATION_USERNAME=<Your Red Hat subscription username>
+  export TPA_SINGLE_NODE_REGISTRATION_PASSWORD=<Your Red Hat subscription password>
+- For Red Hat image registry define :
+  export TPA_SINGLE_NODE_REGISTRY_USERNAME=<Your Red Hat image registry username>
+  export TPA_SINGLE_NODE_REGISTRY_PASSWORD=<Your Red Hat image registry password>
+
+# Todo - Remove ?
+
+Alternatively vagrant will prompt for the registration username and password.
+
+Provide TLS certificates :
```

Comment:
Add in the development section an example about how to create and where place or a file to be loaded automatically by vagrant

Reply:
Yes absolutely.

`kubectl` can easily generate TLS secret manifest files from key and certs.
Meanwhile `kubectl` is not part of vanilla RHEL subscription and we don't want to force end user to hold specific subscriptions in order have that tool available.

The other solution is to install kubernetes repo and pull kubectl from it, but would that be acceptable ?